Zend/zend_virtual_cwd.h

```diff
@@ -98,7 +101,9 @@ typedef unsigned short mode_t;
 #endif
 
 #define IS_SLASH(c)	((c) == '/')
+// IS_SLASH_P() may read the previous char on Windows, which may be OOB; use IS_SLASH_P_EX() instead
 #define IS_SLASH_P(c)	(*(c) == '/')
+#define IS_SLASH_P_EX(c, first_byte)	(*(c) == '/')
```

Comment:
Maybe just this, to not get out of sync:
```suggestion
#define IS_SLASH_P_EX IS_SLASH_P
```

Reply:
Hmm, do you mean
```suggestion
#define IS_SLASH_P_EX(c, first_byte) IS_SLASH_P(c)
```
It seems to me it's important to pass two parameters, even when only building on POSIX, just in case someone builds on Windows.

Reply:
Yeah that's what I meant, sorry